Bootcamp2015-AmazingRace/App.xaml.cs

```diff
@@ -28,11 +28,14 @@ public sealed partial class App
     {
         private WinRTContainer container;
         private SettingsService settings;
+        private static APIService api;
 
+        public static APIService Api{ get{return api;}}
```

Comment:
Doesn't make much sense to make this property static when you've registered it in the container.